doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_nordic_bundles.rst

```diff
@@ -0,0 +1,57 @@
+.. _ug_nrf54h20_suit_nordic_bundles:
+
+Upgrade of Nordic Semiconductor firmware components with SUIT
+#############################################################
+
+Apart from the manufacturer application, the devices also contain the Nordic Semiconductor provided components of the firmware, which are used to provide basic functionalities and security features.
+
+These components are provided in the form of the so called "nRF54h20 SOC binaries", which can be found in https://files.nordicsemi.com/ui/repos/tree/General/SDSC
+
+Updating the Nordic Semiconductor firmware components
+*****************************************************
+
+Updating the Nordic Semiconductor firmware components can be done in two ways.
+Both require using the nordic_top.suit envelope, which can be found inside the Nordic bundle .zip file.
+These two methods are:
+
+* Updating it separately from the manufacturer application
+* Updating it along with the manufacturer application, by attaching the nordic_top.suit envelope to the manufacturer root manifest
+
+Updating the Nordic components separately from the manufacturer application
+===========================================================================
+
+Updating the Nordic components separately from the manufacturer application is done by performing two separate updates.
```

Comment:
To my understanding this sentence is not correct - updating Nordic components is done through a single update, but the update of the whole device (incl. manufacturer application) is done in two steps.

Reply:
After reading that the sentence indeed might sound confusing. Modified that.